gio/src/list_store.rs

```diff
@@ -70,3 +104,17 @@ unsafe extern "C" fn compare_func_trampoline(
         Ordering::Greater => 1,
     }
 }
+
+unsafe extern "C" fn equal_func_trampoline(
```

Comment:
@sdroege Should we move this function into `find_with_equal_func` as well? I thought all callbacks were directly declared inside the block calling them but maybe I'm wrong there...

Reply:
They should unless they are re-used elsewhere

Reply:
Moved callback inside block as it is not re-used elsewhere